xml/storage_multipath.xml

```diff
@@ -4653,11 +4653,11 @@ Give root password for maintenance
     <itemizedlist>
      <listitem>
       <para>
-       <command>chroot</command> to the system root <filename>/</filename> file
-       system from the dracut emergency shell. You may need to mount the
-       <filename>/proc</filename>, <filename>/sys</filename>, and
-       <filename>/dev</filename> partitions as well if dracut needs access to
-       them.
+       Mount the system root <filename>/</filename> file system and
+       <command>chroot</command> to it from the dracut emergency shell. You may
```

Comment:
are people supposed to just know how to do this?
i have done this a few times but following this guide i would have no idea what to do

Reply:
@hramrach, I agree. Imagine a happy user who just accidentally ran into emergency mode for the first time in his life. She needs clear step-by-step instructions.